sql/payday.sql

```diff
@@ -186,13 +186,14 @@ CREATE OR REPLACE FUNCTION process_payment_instruction() RETURNS trigger AS $$
               FROM payday_participants p
              WHERE username = NEW.participant
         );
+
         IF (NEW.amount + NEW.due <= participant.new_balance OR participant.card_hold_ok) THEN
             EXECUTE pay(NEW.participant, NEW.team, NEW.amount + NEW.due, 'to-team');
             RETURN NEW;
-        ELSE
+        ELSIF NEW.amount + NEW.due <= (SELECT minimum_charge FROM settings) THEN
```

Comment:
It looks like the way we [use](https://github.com/gratipay/gratipay.com/blob/6722e5e7d8861588b312b707a3371434c372025d/gratipay/billing/exchanges.py#L169) `MINIMUM_CHARGE`, we should actually make this `<` instead of `<=`. Let's get a test or two for the edge cases here ...